sdk/database/dbplugin/plugin.go

```diff
@@ -43,6 +43,8 @@ type Database interface {
 	// RotateRootCredentials is triggered by a root credential rotation call to
 	// the API.
 	RotateRootCredentials(ctx context.Context, statements []string) (config map[string]interface{}, err error)
+	GenerateCredentials(ctx context.Context) (string, error)
+	SetCredentials(ctx context.Context, statements Statements, staticConfig StaticUserConfig) (username string, password string, err error)
```

Comment:
Can you add comments to these functions?

Reply:
Done in 01dab21e5